src/modules/commander/Commander.cpp

```diff
@@ -578,7 +578,7 @@ Commander::Commander() :
 
 Commander::~Commander()
 {
-	delete[] _airspeed_fault_type;
+
```

Comment:
You can delete this entirely, then in the header set the destructor to `= default`.

Reply:
done